src/spec/integration/importXlsx.spec.coffee

```diff
@@ -169,7 +169,7 @@ describe 'Import integration test', ->
         im.import(csv)
       .then (result) =>
         expect(_.size result).toBe 1
-        expect(result[0]).toBe '[row 3] Product updated.'
+        expect(result[0]).toBe '[row 2] Product updated.'
```

Comment:
I changed this because if you import product with multiple variants mapped by variantId and productId, it will print out the first row `[row 2] Product updated.` but if these variants are mapped by SKU, it is printing out the last row instead `[row 3] Product updated.`.